core/profiles/src/test/java/software/amazon/awssdk/profiles/ProfileFileTest.java

```diff
@@ -326,8 +326,8 @@ public void invalidPropertyNamesAreIgnored() {
 
     @Test
     public void allValidProfileNameCharactersAreSupported() {
-        assertThat(configFileProfiles("[profile ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_]"))
-            .isEqualTo(profiles(profile("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_")));
+        assertThat(configFileProfiles("[profile ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.//%@]"))
+            .isEqualTo(profiles(profile("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.//%@")));
```

Comment:
Nit: Duplicate /

Reply:
I had added it because to future-proof against potentially adding C-style comments `//` in the future. However, on reflection, that's pretty unlikely, and that's not obvious from the test name. I've removed the duplicate in 7b1e1f60922b6d6b5e2810252ce09072bb6c9534.

Reply:
Ah, smart thinking. If you're worried about that, we can add a separate test, but I don't think that's an approval blocker for this change.